src/migration-wizard/contexts/discovery-sources/Provider.tsx

```
@@ -46,16 +48,22 @@ export const Provider: React.FC<PropsWithChildren> = (props) => {
   const [createSourceState, createSource] = useAsyncFn(
-    async (name: string, sshPublicKey: string, httpProxy:string, httpsProxy:string, noProxy: string) => {
+    async (
+      name: string,
+      sshPublicKey: string,
+      httpProxy: string,
+      httpsProxy: string,
+      noProxy: string,
+    ) => {
       try {
         // Build the sourceCreate object conditionally
         const sourceCreate: any = { name };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)